File: sdkwork-app-sdk-csharp/Models/FavoriteFolderCreateForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class FavoriteFolderCreateForm
    {
        public string? FolderName { get; set; }
        public string? Name { get; set; }
        public string? ParentFolderId { get; set; }
        public string? FolderDescription { get; set; }
    }
}

File: sdkwork-app-sdk-go/types/favorite_folder_create_form.go
package types

// 创建收藏文件夹请求
type FavoriteFolderCreateForm struct {
	FolderName string `json:"folderName"`
	Name string `json:"name"`
	ParentFolderId string `json:"parentFolderId"`
	FolderDescription string `json:"folderDescription"`
}
